packages/cfd/src/features/types/containers.types.ts

```diff
@@ -0,0 +1,376 @@
+import { TCFDPlatform, TTradingPlatformAvailableAccount, TTokens, TAccountCategory } from './shared.types';
+import { CATEGORY } from 'Helpers/cfd-config';
+
+type TCFDPasswordFormValues = { password: string };
+
+export type TCFDPersonalDetailsContainerProps = {
+    onSubmit: (index: number, value: { [key: string]: string }) => void;
+};
+
+// cfd-change-password-confirmation
+export type TCFDChangePasswordConfirmationProps = {
+    platform: string;
+    className?: string;
+    context?: RootStore;
+    onCancel: () => void;
+    confirm_label?: string;
+    onConfirm: (values: TCFDPasswordFormValues, actions: FormikActions<TCFDPasswordFormValues>) => void;
+};
+
+export type TCFDDashboardContainer = {
+    active_index: number;
+    platform: TCFDPlatform;
+    is_dark_mode_on: boolean;
+    dxtrade_tokens: TTokens;
+    ctrader_tokens: TTokens;
+};
+
+export type TMT5AccountOpeningRealFinancialStpModal = {
+    enableApp: () => void;
+    disableApp: () => void;
+    toggleCFDVerificationModal: () => void;
+    is_cfd_verification_modal_visible: boolean;
+};
+
+export type TMissingRealAccount = {
+    platform: TCFDPlatform;
+    onClickSignup: () => void;
+};
+
+export type TChangePassword = {
+    platform: TCFDPlatform;
+    onConfirm: () => void;
+};
+
+export type TPasswordResetAndTradingPasswordManager = {
+    email: string;
+    platform: TCFDPlatform;
+    toggleModal?: () => void;
+    account_group: TCFDPasswordReset['account_group'];
+};
+
+export type TResetPasswordIntent = {
+    is_eu: boolean;
+    context?: RootStore;
+    children({ ...props }): React.ReactElement;
+    current_list: Record<string, DetailsOfEachMT5Loginid>;
+};
+
+export type TError = {
+    message: string;
+    code: string | number;
+};
+
+export type TCFDResetPasswordModal = {
+    platform: TCFDPlatform;
+};
+
+export type TCFDPasswordSuccessMessage = {
+    is_investor: boolean;
+    toggleModal: () => void;
+};
+
+export type TFormValues = Record<'old_password' | 'new_password' | 'password_type', string>;
+
+export type TPasswordManagerModalFormValues = Record<'old_password' | 'new_password' | 'password_type', string>;
+
+export type TMultiStepRefProps = {
+    goNextStep: () => void;
+    goPrevStep: () => void;
+};
+
+export type TInvestorPasswordManager = {
+    toggleModal: () => void;
+    error_message_investor: string;
+    is_submit_success_investor: boolean;
+    setPasswordType: (value: string) => void;
+    onSubmit: (values: TPasswordManagerModalFormValues) => Promise<void>;
+    multi_step_ref: React.MutableRefObject<TMultiStepRefProps | undefined>;
+    validatePassword: (values: { old_password: string; new_password: string; password_type: string }) => void | object;
+};
+
+export type TCountdownComponent = {
+    count_from: number;
+    onTimeout: () => void;
+};
+
+export type TCFDPasswordReset = {
+    server: string;
+    account_type: TAccountCategory;
+    password_type: string;
+    account_group: 'demo' | 'real' | '';
```

Comment:
Account group can be either `real` or `demo` empty string should not be the case.
I know we have messed up types in stores but should this source of truth be aligned so we fix it in the components. Otherwise it will be ignored when we try to use these types in components.